interface/portraits/portraits_highelf.gfx

```diff
@@ -23,11 +23,11 @@ spriteTypes = {
 			"GFX_highelf_male_cheeks:d4"
```

Comment:
Problem with eyebrows. The front eyebrow should pass through the hood. The back should go behind the hood.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81292829-59cadc00-907d-11ea-9122-f9bf9e08921d.png)

</details>

Reply:
Nice ?
![image](https://user-images.githubusercontent.com/58479160/81294176-ef6b6900-9087-11ea-9d58-33c574efa378.png)

Reply:
how do the ears actually pierce through the hood?^^

Reply:
> how do the ears actually pierce through the hood?^^

The elves make earholes.